ayon_server/actions/execute.py

```diff
@@ -95,7 +243,150 @@ async def get_launcher_action_response(
+            payload=payload,
+        )
+
+    async def get_redirect_response(
+        self,
+        uri: str,
+        new_tab: bool = False,
+        message: str | None = None,
+        success: bool = True,
+        extra_copy: str | None = None,
+        extra_download: str | None = None,
+    ) -> ExecuteResponseModel:
+        """Return a response for a redirect action"""
+        payload: dict[str, Any] = {
+            "uri": uri,
+            "new_tab": new_tab,
+        }
+        if extra_copy:
+            payload["extra_copy"] = extra_copy
+        if extra_download:
+            payload["extra_download"] = extra_download
+
+        return ExecuteResponseModel(
+            success=success,
+            type="redirect",
+            message=message,
+            payload=payload,
+        )
+
+    async def get_query_response(
+        self,
+        query: dict[str, str | int | float | bool],
+        message: str | None = None,
+        success: bool = True,
+        extra_copy: str | None = None,
+        extra_download: str | None = None,
+    ) -> ExecuteResponseModel:
+        """Return a response for a query action"""
+        payload: dict[str, Any] = {
+            "query": query,
+        }
+        if extra_copy:
+            payload["extra_copy"] = extra_copy
+        if extra_download:
+            payload["extra_download"] = extra_download
+        return ExecuteResponseModel(
+            success=success,
+            type="query",
+            message=message,
+            payload={
+                "query": query,
+            },
+        )
+
+    async def get_form_response(
+        self,
+        title: str,
+        fields: SimpleForm,
+        submit_label: str | None = "Submit",
+        submit_icon: str = "check",
+        cancel_label: str | None = "Cancel",
+        cancel_icon: str = "close",
+        message: str | None = None,
+        success: bool = True,
+        extra_copy: str | None = None,
+        extra_download: str | None = None,
+    ) -> ExecuteResponseModel:
+        """Return a response for a form action"""
+        payload = {
+            "title": title,
+            "fields": list(fields),
+            "submit_label": submit_label,
+            "submit_icon": submit_icon,
+            "cancel_label": cancel_label,
+            "cancel_icon": cancel_icon,
+        }
+        if extra_copy:
+            payload["extra_copy"] = extra_copy
+        if extra_download:
+            payload["extra_download"] = extra_download
+        return ExecuteResponseModel(
+            success=success,
+            type="form",
+            message=message,
+            payload=payload,
+        )
+
+    #
+    # Deprecated methods
+    #
+
+    async def get_launcher_action_response(
+        self,
+        args: list[str],
+        message: str | None = None,
+    ) -> ExecuteResponseModel:
+        """Deprecated alias for get_launcher_response"""
+        return await self.get_launcher_response(
```

Comment:
I like to use `warnings.warn` as it prints out message 🙂 .
```suggestion
        warnings.warn(
            "Used deprecated method 'get_launcher_action_response'.",
            DeprecationWarning
        )
        return await self.get_launcher_response(
```

Reply:
And similar to `get_server_action_response`.